modules/nw-about-static-egress-ips.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-static-egress-ips.adoc
+
+[id="about-{context}"]
+= About static IPs for external project traffic
+
+As a cluster administrator you can assign specific, static IP
+addresses to projects, so that traffic is externally easily recognizable. This
+is different from the default egress router, which is used to send traffic to
```

Comment:
"from the egress router" or "from egress routers". There's no "default" egress router. (This is comparing the egress IP feature to the egress router feature, https://docs.okd.io/latest/admin_guide/managing_networking.html#admin-guide-limit-pod-access-egress-router.)